sale_commission_so_based/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This module provides the external functionality for sale_commission_salesman operations to enable the
```

Comment:
question: how is module sale_commission_salesman related to this module? I don't see it as a dependency

Reply:
@francesco-ooops You are right, typo in the description. Thank's